maps/tradeship/tradeship-2.dmm

```diff
@@ -3076,9 +3075,6 @@
 /area/ship/trade/hidden)
 "hm" = (
 /obj/item/stool/padded,
-/obj/machinery/atmospherics/pipe/simple/hidden/supply{
```

Comment:
Were these removed pipes just not used anywhere?

Reply:
These pipes are isolated: they connect to no other pipes. They used to be deleted during `atmos_init` (prior to unit tests, after atom/Initialize), but no longer are: now they spawn in and are "leaking," failing the no leaking pipes unit test.